addon/components/docs-keyboard-shortcuts/index.js

```diff
@@ -22,9 +22,9 @@ export default class DocsKeyboardShortcutsComponent extends Component {
   @onKey('KeyG', { event: 'keyup' })
   goto() {
     if (!formElementHasFocus()) {
-      this.set('isGoingTo', true);
+      this.isGoingTo = true;
```

Comment:
Does `isGoingTo` need to be a tracked prop?

Reply:
I don't think so. 

My understanding is that `isGoingTo` is basically just an internal flag that we're using to chain together multiple keyup events for a keyboard shortcut rather than something that affects how the component is rendered.